cache/fast_lru_cache.h

```diff
@@ -307,6 +302,7 @@ std::shared_ptr<Cache> NewFastLRUCache(
     size_t capacity, int num_shard_bits = -1,
     bool strict_capacity_limit = false,
     CacheMetadataChargePolicy metadata_charge_policy =
-        kDefaultCacheMetadataChargePolicy);
+        kDefaultCacheMetadataChargePolicy,
+    size_t estimated_charge_per_entry = 1);
```

Comment:
Its probably better to force the caller to provide the estimated charge, rather than default to 1 which may be wildly inaccurate.